react/features/unsupported-browser/components/UnsupportedMobileBrowser.js

```diff
@@ -82,10 +83,21 @@ class UnsupportedMobileBrowser extends Component<*, *> {
         // appears to be a link with an app-specific scheme, not a Universal
         // Link.
         const appScheme = interfaceConfig.MOBILE_APP_SCHEME || 'org.jitsi.meet';
-        const joinURL = `${appScheme}:${window.location.href}`;
+        const joinURL = parseURIString(window.location.href);
+
+        // Drop the protocol only if it's the https, because that's the app's
+        // default scheme which will be eventually added later on when
+        // the intent URI is being processed. We'd like to preserve the protocol
```

Comment:
Where does the intent URI get processed? It doesn't feel good to have special handling just for the https case so I'd like to know more about the context.

Reply:
So everything that is after the app scheme part (org.jitsi.meet) goes to the mobile app and then that remainder part (I think) gets set as the 'url' property of the AbstractApp. Parsing of the URL happens in the uri.js referenced here as well. In case of https: it would end up being added here: https://github.com/jitsi/jitsi-meet/blob/master/react/features/base/util/uri.js#L99

Reply:
Do you think it's better to just strip the protocol part given WebRTC does no longer work with http sites (on Chrome at least)?

Reply:
So the issue is that https: was getting appended twice and what this fix is making sure that you don't append https:, but let everything else go through. Why are you worried about supporting http for the mobile case? Does mobile webrtc really not work when you've joined a conference hosted on http? It makes sense to me to only support https to be honest... Or at least is this logic that should be done.

Reply:
The issue was that the other mobile app consumer wants a clean URI passed to the app as an intent (it does not care about the http). Before the changes the intent URI would be:
org.jitsi.meet://http://server.com/room or org.jitsi.meet://https://server.com/room
After the changes (in both cases):
org.jitsi.meet://server.com/room
Only after the changes the default logic would kick in because there was no protocol and https would be added.
So what I was trying to achieve here was avoid loosing the http part of the information.
But I will just remove this part.